### ISSUE TITLE
Problem when resolving domain name

### ISSUE THREAD

**Issue by crdffrance** [LJPc-solutions/Pure-PHP-DoH-Client#1]
Hello,

Thanks for this great script!

When I try to resolve the domain name "000-602staging-files-0readme.00-1-okta-mfa-ackorigin---0-8-0ing0-01eth0houseroomdev2-4-8logs.bcgfed.com", the script returns an error.

Here is the original code:

```
$dnsr = DNS::query('000-602staging-files-0readme.00-1-okta-mfa-ackorigin---0-8-0ing0-01eth0houseroomdev2-4-8logs.bcgfed.com', DNSType::{$qtype}() );
$dns[$qtype] = $dnsr->getAnswers();
```

This is the error returned :

```
PHP Warning:  unpack(): Type n: not enough input, need 2, have 0 in /root/***/vendor/ljpc/pure-php-doh-client/src/DNS.php on line 136
PHP Fatal error:  Uncaught TypeError: Cannot assign bool to property LJPc\DoH\DNS::$header of type array in /root/ai_php/vendor/ljpc/pure-php-doh-client/src/DNS.php:136
Stack trace:
#0 /root/ai_php/vendor/ljpc/pure-php-doh-client/src/DNS.php(48): LJPc\DoH\DNS::processHeader()
#1 /root/ai_php/parse.php(147): LJPc\DoH\DNS::query()
#2 {main}
  thrown in /root/ai_php/vendor/ljpc/pure-php-doh-client/src/DNS.php on line 136
```

But the domain name does a lot of things:

```
dig A 000-602staging-files-0readme.00-1-okta-mfa-ackorigin---0-8-0ing0-01eth0houseroomdev2-4-8logs.bcgfed.com

; <<>> DiG 9.18.19-1~deb12u1-Debian <<>> A 000-602staging-files-0readme.00-1-okta-mfa-ackorigin---0-8-0ing0-01eth0houseroomdev2-4-8logs.bcgfed.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 3826
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 1232
;; QUESTION SECTION:
;000-602staging-files-0readme.00-1-okta-mfa-ackorigin---0-8-0ing0-01eth0houseroomdev2-4-8logs.bcgfed.com. IN A

;; ANSWER SECTION:
000-602staging-files-0readme.00-1-okta-mfa-ackorigin---0-8-0ing0-01eth0houseroomdev2-4-8logs.bcgfed.com. 14400 IN A 165.160.13.20
000-602staging-files-0readme.00-1-okta-mfa-ackorigin---0-8-0ing0-01eth0houseroomdev2-4-8logs.bcgfed.com. 14400 IN A 165.160.15.20

;; Query time: 36 msec
;; SERVER: 1.1.1.1#53(1.1.1.1) (UDP)
;; WHEN: Wed Oct 18 12:52:32 CEST 2023
;; MSG SIZE  rcvd: 164

```

How can I correct this problem?

Thank you,

Regards,

**Comment by Lars-** [comment]
Interesting! It probably has something to do with the large query, but I'm not sure yet. I will let you know when I find out what the problem is.

**Comment by crdffrance** [comment]
> Interesting! It probably has something to do with the large query, but I'm not sure yet. I will let you know when I find out what the problem is.

Thank you :)

**Comment by Lars-** [comment]
Fixed in 1.3.0
